api/envoy/admin/v2alpha/memory.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package envoy.admin.v2alpha;
+
+// [#protodoc-title: Memory]
+
+// Proto representation of the internal memory consumption of an Envoy instance.
+message Memory {
+
+  // The number of bytes allocated by the heap for Envoy.
+  uint64 currently_allocated_bytes = 1;
```

Comment:
Do you want to provide references back to the tcmalloc docs for folks who want to understand some of these metrics more deeply?

Reply:
Sure, I will link them to http://gperftools.github.io/gperftools/tcmalloc.html and try to standardize the field names everywhere.